pkg/utils/nodepool/nodepool.go

```diff
@@ -129,3 +129,14 @@ func OrderByWeight(nps []*v1.NodePool) {
 		return weightA > weightB
 	})
 }
+
+func CopyOfferings(offerings cloudprovider.Offerings) []*cloudprovider.Offering {
```

Comment:
Should this be in this package? It seems like the cloudprovider package is more appropriate.